src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -75,6 +75,9 @@ public ICollection<string> Exclude
         [Option(typeof(NuGetCommand), "CommandMSBuildVersion")]
         public string MSBuildVersion { get; set; }
```

Comment:
Can we warn when the user passes both `MSBuildVersion` and `MSBuildPath` stating that the `MSBuildPath` is preferred.